### PR TITLE
Bump eks hashicups

### PR DESCRIPTION
**Pull request by jjtimmons**
- new hashicups has a cooler, more detailed UI
- [nginx got removed from the frontend](https://github.com/hashicorp-demoapp/frontend/issues/35), so moved to its own service
- 8080 -> 80, default port for http
- bumped consul helm version (there was a bug hindering nginx image deployment)
- small change to ecs log-group (should be a different PR) to avoid duplicates

### Testing

opened app, made an order

Screen Shot 2022-03-21 at 9 21 26 PM

### CODE REVIEW

**Comment by jjtimmons** [comment]
@i0rek I see the module version file but didn't bump to 0.6.2 (despite changes to 2 modules). I assume I should bump + apply that script?

**Review by i0rek — APPROVED** [review]
The changes look good to me! Great job!

There is some automation that ensures tests are up to date, every example uses the same hashicups version and templates are updated. Running `make` will regenerate a whole bunch of things. One thing that will happen since you updated `hashicups_version.sh` is that hashicups versions will be set to this across every example. Including the ecs example that we didn't upgrade, so it would be left in an unknown state. Which wouldn't be a big problem since terraform registry only updates the module when a new version is tagged. 

Basically up to you. We can merge this, fix ecs and then run `make` to make sure everything is in order.

**Comment by i0rek** [comment]
Our tooling relies on `intentions.tf` to be there (even if its empty):

https://github.com/hashicorp/terraform-aws-hcp-consul/blob/d1b2c0060e05ccb10284936635b5b2e6c0981693/scripts/generate_ui_templates.sh#L4

I understand that it is a bit strange to have an empty file. We could change the tooling to not rely on it being present.

**Comment by jjtimmons** [comment]
Thanks for the review. I added back the `intentions.tf` file. I also poked briefly at ECS and think that the bump there may be non-trivial (because of same nginx issue/breaking change in the frontend), so I'm leaving towards your recommended second approach:

> We can merge this, fix ecs and then run make to make sure everything is in order.
